admin_guide/persistent_storage_nfs.adoc

```diff
@@ -84,3 +84,18 @@ reclamation of discarded resources can be configured for each persistent volume.
 Many ways to script these tasks exist.  We've created an link:https://github.com/openshift/openshift-ansible/tree/master/roles/kube_nfs_volumes[Ansible playbook] to help you get started.
 
 
+== SELinux and NFS export settings
+
+By default, SELinux will disallow a connection from a pod to a remote NFS server.  Enable this in SELinux on each node with the following command:
+
+```
+setsebool virt_use_nfs 1
+```
+
+Additionally, each exported volume on the NFS server itself must conform to the following:
+
+* each export must be: ```/example_fs *(rw,no_root_squash)```
```

Comment:
While * is simplest to tell people, without knowledge of their networking, we should note that it is fine to restrict it to the range of node hosts that could possibly be using the storage. Might be best just to give an example and refer to the `/etc/exports` man page.

I found that no_root_squash was not sufficient in the mysql case - it got confused by the ownership of the volume itself. I'm not certain of the best recommendation here but what seemed to work was `all_squash`.

So I'd say something like:

> To enable use by applications with arbitrary user permissions, each export should have the permissions `(rw,all_squash)`, e.g. in `/etc/exports`:
> 
>         /example_fs  *(rw,all_squash)
> 
> Refer to the `/etc/exports` man page for further details of defining exports, e.g. to restrict access to only node hosts.